cli/src/seth_cli/client/client.go

```diff
@@ -93,7 +96,8 @@ func (c *Client) Get(address []byte) (*EvmEntry, error) {
 	}
 
 	if len(buf) == 0 {
-		return nil, nil
+		err := fmt.Errorf("received buffer length is zero")
```

Comment:
This error message could be improved to add more detail about what went wrong exactly